<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>producer</class>
 <widget class="QWidget" name="producer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>申论生成器</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_start">
       <property name="font">
        <font>
         <family>华文仿宋</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>开头:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>仿宋</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>请输入开头的一句话</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_num">
       <property name="font">
        <font>
         <family>华文仿宋</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>字数:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="font">
        <font>
         <family>仿宋</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>仿宋</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>生成</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTextBrowser" name="resultText">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>1677215</width>
       <height>1677215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>此处是生成器的内容</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>producer</receiver>
   <slot>ui_start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1144</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>1332</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ui_start()</slot>
 </slots>
</ui>
